Scripts/Core/EditorVR.DirectSelection.cs

```diff
@@ -13,9 +13,10 @@ partial class EditorVR
 		class DirectSelection : Nested
 		{
 			internal IGrabObjects objectsGrabber { get; set; }
-
+			internal Dictionary<Transform, DirectSelectionData> directSelections { get { return m_DirectSelections; } }
+			
 			// Local method use only -- created here to reduce garbage collection
-			readonly Dictionary<Transform, DirectSelectionData> m_DirectSelectionResults = new Dictionary<Transform, DirectSelectionData>();
+			readonly Dictionary<Transform, DirectSelectionData> m_DirectSelections = new Dictionary<Transform, DirectSelectionData>();
```

Comment:
This is no longer just to reduce garbage collections, but now also holds the results, so move it above this area.